test/integration/integration_test.go

```diff
@@ -81,8 +81,8 @@ func FeatureContext(s *godog.Suite) {
 
 	// steps to execute minishift commands
 	s.Step(`Minishift (?:has|should have) state "([^"]*)"`, minishift.shouldHaveState)
+	s.Step(`executing "minishift ([^"]*)" (succeeds|fails)$`, executingMinishiftCommandSucceedsOrFails)
```

Comment:
Order should not matter. If it does we do something wrong. We might for example need to match the end of line in '`executing "minishift ([^"]*)"`'